Docs/source/running_cpp/parameters.rst

```diff
@@ -1873,6 +1873,20 @@ Reduced Diagnostics
             the macroparticle weight will be used to compute
             the histogram, and no normalization will be done.
 
+        * ``<reduced_diags_name>.filter_function(t,x,y,z,ux,uy,uz)`` (`string`) optional
+            Users can provide an expression returning a boolean for whether a particle is taken
+            into account when calculating the histogram (the exact test is whether the return
+            value is `> 0.5`).
+            `t` represents the physical time in seconds during the simulation.
+            `x, y, z` represent particle positions in the unit of meter.
+            `ux, uy, uz` represent particle velocities in the unit of
+            :math:`\gamma v/c`, where
+            :math:`\gamma` is the Lorentz factor,
+            :math:`v/c` is the particle velocity normalized by the speed of light.
```

Comment:
It's probably most consistent if we convert those to `beta*gamma` as in the other I/O places.
@atmyers shows how to do this in  #1598, essentially doing a 
```C++
pc->ConvertUnits(ConvertDirection::WarpX_to_SI);
```
before the filter and a
```
pc->ConvertUnits(ConvertDirection::SI_to_WarpX);
```
afterwards.

Instead of `SI` we would put `WarpX` in both places, I think.

Reply:
I think that it's already in units of `beta*gamma` (as in the other functions), isn't it?

If I'm not mistaken, I think the `pc->ConvertUnits()` functions are here because we dump the full diagnostics in SI units, but I don't think we need them here because we don't do that for the `ParticleHistogram` reduced diag (the momenta in the histogram function are given in units of `beta*gamma`).

Reply:
Oh right, since we store proper velocity on u (v*gamma), the v/c is beta*gamma :)

Should we document this here as such? I think it's not clear to readers that `v` stands for proper velocity when they read this line.

Reply:
By here, you mean on the documentation lines above? If so, `v` here is the standard velocity (not the proper velocity): 

```
`ux, uy, uz` represent particle velocities in the unit of :math:`\gamma v/c`
```

(there's already a sneaky `\gamma` :) )

Reply:
lol, I cannot read. there is a gamma, all clear :rofl: